bugout-dev/moonstream
# Add ETHTXPOOL_HUMBUG_CLIENT_ID as env variable for escape desynchroni…

…ze with crawler.




## Changes

Add read client_id from envioment as tx_pool crawler do it. 




## How to test these changes?
Check if after change ETHTXPOOL_HUMBUG_CLIENT_ID crawler put correct tag and bugout provider read it.




## Related issues
